<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Firma</class>
 <widget class="QMainWindow" name="Firma">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firma</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1011</width>
      <height>531</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_prehled">
     <attribute name="title">
      <string>Prehled</string>
     </attribute>
     <widget class="QListWidget" name="lw_novi">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>201</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_n_zamitnout">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Zamitnout</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_n_dale">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Dale</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>15</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Novi</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Schvalen</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Prijat</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Zamintut</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lw_schvaleni">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>201</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="lw_prijati">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>60</y>
        <width>201</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="lw_zamitnuti">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>60</y>
        <width>201</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_s_zamitnout">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Zamitnout</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_s_dale">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Dale</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_detail">
     <attribute name="title">
      <string>Detail</string>
     </attribute>
     <widget class="QLabel" name="l_obrazek">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="l_jmeno">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Jmeno Prijmeni</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_vek">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>vek</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_bydliste">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>130</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>bydliste</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_schopnosti">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>180</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>schopnosti</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_stav">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>40</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>STAV</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1029</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
